CenterForOpenScience/osf.io
# [PLAT-1012] Custom citations

## Purpose

Creating custom citations is a frequently requested feature, lets do it!

## Changes

- Add citations serializers 
- Add new logging for citations editing
- Add tests
- Add front-end elements

![screen shot 2018-08-30 at 12 07 03 pm](https://user-images.githubusercontent.com/9688518/44864295-5139c580-ac4d-11e8-88b0-44e5a2817039.png)

## QA Notes

key features:

- add/remove/edit citations from project overview page
- all the above should appear in log

## Documentation

Will require minor dev docs change when done and probably help site changes.

## Side Effects

None that I know of.

## Ticket

https://openscience.atlassian.net/projects/PLAT/issues/PLAT-1012
